pkg/snapshot/snapshot.go

```diff
@@ -170,7 +169,7 @@ func (s *Snapshotter) scanFullFilesystem() ([]string, []string, error) {
 	filesToAdd := []string{}
 	for path := range memFs {
 		if util.CheckWhitelist(path) {
-			logrus.Tracef("Not adding %s to layer, as it's whitelisted", path)
+			logrus.Infof("Not adding %s to layer, as it's whitelisted", path)
```

Comment:
Maybe debug? There are a lot of whitelisted files iirc

Reply:
sorry not an intentional change.